### PR TITLE
Adding ADO build pipeline

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -79,12 +79,23 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     };
 
     templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+    templatizeTree(locals, path.resolve(sourceDir, "utilities"), path.resolve(apiOutputDir, "utilities"));
```
Comment (jplafonta) [inline make.js:82]: >path.resolve(apiOutputDir, "utilities") [](http://example.com/codeflow?start=65&length=39)

does this work if the utilities directory doesn't already exist in the output repo?

File: source/build/Win32/PlayFabCore.Win32.vcxproj.filters.ejs [hunk 2]
```diff
@@ -57,9 +57,6 @@ for(var featureGroupName in SDKFeatureGroups) { %>
     </ClCompile><%
     }
 } %>
-    <ClInclude Include="$(PlayFabSourceDir)\source\stdafx.h" >
-      <Filter>Source</Filter>
-    </ClInclude>
```
Comment (jplafonta) [inline source/build/Win32/PlayFabCore.Win32.vcxproj.filters.ejs:62]: intentionally removed?

**Review by jplafonta — APPROVED** [review]
:shipit:

File: source/code/source/Common/ApiAsyncProviders.h.ejs [hunk 3]
```diff
@@ -151,6 +151,7 @@ protected:
     template<typename T>
     HRESULT GetResultHelper(void* buffer, size_t bufferSize, const T& result)
     {
+        UNREFERENCED_PARAMETER(bufferSize);
```
Comment (jplafonta) [inline source/code/source/Common/ApiAsyncProviders.h.ejs:154]: >     [](http://example.com/codeflow?start=43&length=4)

nit: remove trailing spaces

File: make.js (hunk 1 above)
Comment (jasonsandlin) [reply make.js:82]: yeah

**Pull request merged**